pkg/api/admin/operatorflags.go

```diff
@@ -0,0 +1,54 @@
+package admin
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+type operatorFlagsMergeStrategyStruct struct {
+	OperatorFlagsMergeStrategy string
+	Oc                         api.OpenShiftCluster
+}
+
+const operatorFlagsMergeStrategyDefault string = "merge"
```

Comment:
Is that better to replace the string literals "merge" and "reset" with constants, especially if they are used in multiple places in your codebase to prevent bugs due to typos?
```
const (
	OperatorFlagsMergeStrategyDefault = "merge"
	OperatorFlagsMergeStrategyMerge   = "merge"
	OperatorFlagsMergeStrategyReset   = "reset"
)
```

Reply:
Agreed, thats better, change made.